python/semantic_kernel/connectors/ai/content_filter_ai_exception.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Dict, Optional
+
+from semantic_kernel.connectors.ai.ai_exception import AIException
+
+
+class ContentFilterAIException(AIException):
```

Comment:
is this specific to OpenAI, Azure OpenAI, or are their similar things in other LLM services? If not this file should be somewhere within the opena_ai folders.

Reply:
As of now, yes, this is specific to Azure OpenAI.

Reply:
Moved it to open_ai folder